### PR TITLE
CRM-19743 Ensure that checking for id in sort column works in arrays …

### PR DESCRIPTION
**Pull request by seamuslee001**
…as well as strings

### CODE REVIEW

File: tests/phpunit/api/v3/SyntaxConformanceTest.php [hunk 1]
```diff
@@ -1059,6 +1059,24 @@ public function testInvalidSort_get($Entity) {
    * @param $Entity
    * @throws \PHPUnit_Framework_IncompleteTestError
    */
+  public function testValidSortSingleArrayById_get($Entity) {
+    $invalidEntitys = array('ActivityType', 'Setting', 'System');
+    if (in_array($Entity, $invalidEntitys)) {
+      $this->markTestSkipped('It seems OK for ' . $Entity . ' to skip here as it silently sips invalid params');
+    }
+    $params = array('sort' => array('id'));
```
Comment (eileenmcnaughton) [inline tests/phpunit/api/v3/SyntaxConformanceTest.php]: Sorry the array with the string array('id DESC'))

**Comment by adixon** [comment]
If this is only applying to 4.6, we need another one for 4.7/master, the error is the same there, even if the specific symptom doesn't show up. I'll resurrect my original one if you need a separate pull request.

**Comment by seamuslee001** [comment]
@adixon master version of this PR has been merged https://github.com/civicrm/civicrm-core/pull/9525 just need someone to merge this one

**Comment by seamuslee001** [comment]
as #9525 been merged i think we should merge this @eileenmcnaughton

**Pull request merged**